src/python/bot/fuzzers/syzkaller/engine.py

```diff
@@ -151,7 +151,8 @@ def fuzz(self, target_path, options, unused_reproducers_dir=None, max_time=0):
     self._create_temp_corpus_dir('new')
 
     args = options.arguments
-    args += ['--coverfile', runner.get_cover_file_path()]
+    if not environment.is_android_cuttlefish():
+      args += ['--coverfile', runner.get_cover_file_path()]
```

Comment:
Add TODO here to remove this and implement something to call the HTTP endpoint ourselves?

Reply:
Done